web/src/routes/PoisPage.tsx

```diff
@@ -44,14 +43,12 @@ const moveViewToBBox = (bBox: BBox, defaultVp: MapViewViewport): MapViewViewport
   ])
 }
 
-type PropsType = CityRouteProps & RouteProps<typeof POIS_ROUTE>
```

Comment:
Is `RouteProps<typeof XX_ROUTE>` no longer working with the update or why have you decided to remove/change it?

Reply:
Yes, no longer working as this was also a path-to-regexp utility. Also, routes are now relative so `cityCode` and `languageCode` are not even included in `POIS_ROUTE` and so on.